roles/agnosticv/templates/governor.yml.j2

```diff
@@ -66,11 +66,9 @@ spec:
 {% endfor %}
 
   actions:
-{% for __action in anarchy_governor_standard_actions
- | union(_entry_points.keys() | list)
-%}
-{#   Skip action if entry point is explicitly set to null or empty string. #}
-{%   if __action not in _entry_points or _entry_points[__action] | default('', true) != '' %}
+{% for __action, __action_config in _actions.items() %}
+{#   Skip action if disabled or entry point is explicitly set to null or empty string. #}
+{%   if not __action_config.disabled | default(false) boolean and (__action not in _entry_points or _entry_points[__action] | default('', true) != '') %}
```

Comment:
Can we take the logic out of the template?
Maybe define the var outside, and keep the test in the template as simple as possible.

Instead of going through a list and then test for each item inside the template, which one should be added, in the "code" that calls the template we built the final list, and we don't have any test in the template?

Reply:
I'll see what I can come up with...